packages/fury-adapter-oas3-parser/lib/parser/oas/parseMediaTypeObject.js

```diff
@@ -19,9 +20,49 @@ const unsupportedKeys = ['encoding'];
 const isUnsupportedKey = R.anyPass(R.map(hasKey, unsupportedKeys));
 
 function isJSONMediaType(mediaType) {
-  const type = contentType.parse(mediaType);
-  const jsonMediaType = /^application\/\S*json$/i;
-  return jsonMediaType.test(type.type);
+  const contentType = contentTyper.parse(mediaType);
+  const { type, suffix, subtype } = mediaTyper.parse(contentType.type);
```

Comment:
I've refactored isJSONMediaType to use mediaTyper so it is implemented in the same way that `isTextMediaType` is implemented. I think it can be a bit safer to use mediaTyper than custom regex.